### DISCUSSION TITLE
Will feature "string_view" be merged into master someday?

### DISCUSSION THREAD

**Discussion by lyd405121** [ocornut/imgui#7808] [New Users Build/Link/Run issues ONLY!]
As docking and multi-select has been merged,
Is there any plan to merge string_view?

**Comment by ocornut** [comment]
Yes. Maybe this year.
I mostly worry about debugger stepping overhead.

**Reply by ocornut** [reply]
Docking hasn’t been merged yet btw.
But of course every feature is designed to be merged when finished.
